samples/frameworks/react/ts/src/hooks/useServiceBus.ts

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT license.
+
+  This sample demonstrates how to create a React hook integrating
+  with Azure Service Bus. 
+
+  For more information on Azure Service Bus please see 
+  https://www.npmjs.com/package/@azure/service-bus
+*/
+
+import { credential } from "../utils/auth";
+import {
+  ServiceBusClient,
+  ServiceBusMessage,
+  ServiceBusReceiver,
+  ServiceBusSender
+} from "@azure/service-bus";
+import { useEffect, useRef } from "react";
+import { getEnvironmentVariable } from "../utils";
+
+const useServiceBus: (
```

Comment:
This is definitely not great to read - I was wondering what is the recommended approach for declaring the types of this hook? One thing I could do is name that type instead of inlining the definition, but figured I'll ask folks with more experience first

Reply:
I would refactor the type into its own definition if it will be exported or being used somewhere else. Otherwise, this looks good to me. This is just me though :)